config/html-webpack-inline-critical-css-plugin.js

```diff
@@ -0,0 +1,256 @@
+const fs = require('fs');
+const path = require('path');
+const parse5 = require('parse5');
+const nwmatcher = require('nwmatcher');
+const css = require('css');
+
+const treeUtils = parse5.treeAdapters.htmlparser2;
+
+const PLUGIN_NAME = 'html-webpack-inline-critical-css-plugin';
+
+const PARSE5_OPTS = {
+  treeAdapter: treeUtils
+};
+
+function defineProperties(obj, properties) {
+  for (let i in properties) {
+    let value = properties[i];
```

Comment:
Why are these not `const`?

Reply:
+1 we should only use `let` if the value is changed.

Reply:
whoop, old habits die hard. I imagine there's a lint rule to remind me of this.